### PR TITLE
fix: use defined jupyter port

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingyang@tensorchord.ai>

* not specified: generate a free port
* specified: use it (even if it's not free) (just let it crash)

### CODE REVIEW

**Review by gaocegege — APPROVED** [review]

**Comment by gaocegege** [comment]
Maybe we should remove the JupyterConfig.Port directly in the future.

File: pkg/lang/ir/editor.go [hunk 1]
```diff
@@ -85,7 +86,7 @@ func (g Graph) generateJupyterCommand(workingDir string) []string {
 			"--NotebookApp.token", "''")
 	}
 	if g.JupyterConfig.Port != 0 {
```
Comment (hezhizhen) [inline pkg/lang/ir/editor.go:88]: The `if` statement is not necessary.

Comment (gaocegege) [reply pkg/lang/ir/editor.go:88]: I think so, please help fix this if you are interested.

Comment (hezhizhen) [reply pkg/lang/ir/editor.go:88]: https://github.com/tensorchord/envd/pull/773

**Pull request merged**